typings/web-inspector.d.ts

```diff
@@ -17,7 +17,6 @@ declare global {
       url: string;
       _url: string;
       protocol: string;
-      origin: string | null;
```

Comment:
I know this was from the last PR, but should we put a warning on `origin` to not use it? like
```js
/** use `parsedURL.securityOrigin()` instead */
origin: never;
```
or something. It could probably sneak back in in the future otherwise (just a comment might also work :)

Reply:
yeah good call done